CI build failure: Build MacCatalyst x64 Release AllSubsets_Mono - XcodeBuildApp task failed unexpectedly - Could not find System.Runtime.Tests.app

Hit this in an unrelated PR in `main`: https://github.com/dotnet/runtime/pull/78744

- Queue: `Build MacCatalyst x64 Release AllSubsets_Mono`
- Job results: https://dev.azure.com/dnceng-public/public/_build/results?buildId=91705&view=logs&j=a2a39d6b-70c5-5040-933a-4b8b6e52bf19&t=7de46cbc-684f-5a34-7a8f-d5a9c81d3460&l=3184
- Failed step: `Build product`

```
Running: cmake -S. -BNormalTestAppBundles -GXcode -DCMAKE_SYSTEM_NAME=Darwin -DCMAKE_OSX_ARCHITECTURES=x86_64
  Using working directory: /Users/runner/work/1/s/artifacts/bundles//tests.all/
  Exit code: 0
Running: xcodebuild ONLY_ACTIVE_ARCH=YES CODE_SIGN_IDENTITY="-" -scheme "System.Runtime.Tests" -destination "generic/platform=macOS,name=Any Mac,variant=Mac Catalyst" -UseModernBuildSystem=YES -archivePath "/Users/runner/work/1/s/artifacts/bundles/tests.all/NormalTestAppBundles/System.Runtime.Tests/AppBundle" -derivedDataPath "/Users/runner/work/1/s/artifacts/bundles/tests.all/NormalTestAppBundles/System.Runtime.Tests/AppBundle" IPHONEOS_DEPLOYMENT_TARGET=13.5 -configuration Release
  Using working directory: /Users/runner/work/1/s/artifacts/bundles/tests.all/NormalTestAppBundles/System.Runtime.Tests/AppBundle
  Exit code: 0
/Users/runner/work/1/s/src/libraries/tests.proj(735,5): error MSB4018: The "XcodeBuildApp" task failed unexpectedly.
/Users/runner/work/1/s/src/libraries/tests.proj(735,5): error MSB4018: System.IO.DirectoryNotFoundException: Could not find a part of the path '/Users/runner/work/1/s/artifacts/bundles/tests.all/NormalTestAppBundles/System.Runtime.Tests/AppBundle/Release-maccatalyst/System.Runtime.Tests.app'.
/Users/runner/work/1/s/src/libraries/tests.proj(735,5): error MSB4018:    at System.IO.FileSystem.MoveDirectory(String sourceFullPath, String destFullPath, Boolean isCaseSensitiveRename)
/Users/runner/work/1/s/src/libraries/tests.proj(735,5): error MSB4018:    at System.IO.FileSystem.MoveDirectory(String sourceFullPath, String destFullPath)
/Users/runner/work/1/s/src/libraries/tests.proj(735,5): error MSB4018:    at Xcode.BuildAppBundle(String xcodePrjPath, Boolean optimized, String devTeamProvisioning, String destination) in /_/src/tasks/AppleAppBuilder/Xcode.cs:line 537
/Users/runner/work/1/s/src/libraries/tests.proj(735,5): error MSB4018:    at XcodeBuildApp.Execute() in /_/src/tasks/AppleAppBuilder/Xcode.cs:line 108
/Users/runner/work/1/s/src/libraries/tests.proj(735,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
/Users/runner/work/1/s/src/libraries/tests.proj(735,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)
##[error]src/libraries/tests.proj(735,5): error MSB4018: (NETCORE_ENGINEERING_TELEMETRY=Build) The "XcodeBuildApp" task failed unexpectedly.
System.IO.DirectoryNotFoundException: Could not find a part of the path '/Users/runner/work/1/s/artifacts/bundles/tests.all/NormalTestAppBundles/System.Runtime.Tests/AppBundle/Release-maccatalyst/System.Runtime.Tests.app'.
   at System.IO.FileSystem.MoveDirectory(String sourceFullPath, String destFullPath, Boolean isCaseSensitiveRename)
   at System.IO.FileSystem.MoveDirectory(String sourceFullPath, String destFullPath)
   at Xcode.BuildAppBundle(String xcodePrjPath, Boolean optimized, String devTeamProvisioning, String destination) in /_/src/tasks/AppleAppBuilder/Xcode.cs:line 537
   at XcodeBuildApp.Execute() in /_/src/tasks/AppleAppBuilder/Xcode.cs:line 108
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)

Build FAILED.
```

```json
{
  "ErrorMessage": "The \"XcodeBuildApp\" task failed unexpectedly.",
  "BuildRetry": false
}
```

<!--Known issue error report start -->
### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[98927](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98927)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98927/logs/423)|dotnet/runtime#79090|
|[98915](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98915)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98915/logs/560)|dotnet/runtime#79088|
|[98840](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98840)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98840/logs/530)|dotnet/runtime#74579|
|[98581](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98581)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98581/logs/2216)|dotnet/runtime#79061|
|[97485](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97485)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97485/logs/1306)|dotnet/runtime#78918|
|[98706](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98706)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98706/logs/565)|dotnet/runtime#79076|
|[98698](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98698)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98698/logs/569)|dotnet/runtime#78915|
|[98688](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98688)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98688/logs/385)||
|[98678](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98678)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98678/logs/984)|dotnet/runtime#78852|
|[98671](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98671)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98671/logs/951)||
|[98663](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98663)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98663/logs/887)|dotnet/runtime#78354|
|[98639](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98639)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98639/logs/597)|dotnet/runtime#79074|
|[98594](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98594)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98594/logs/954)|dotnet/runtime#79048|
|[98571](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98571)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98571/logs/601)|dotnet/runtime#79071|
|[98547](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98547)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98547/logs/917)|dotnet/runtime#79069|
|[98494](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98494)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98494/logs/665)|dotnet/runtime#78782|
|[98450](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98450)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98450/logs/333)||
|[98405](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98405)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98405/logs/959)||
|[98388](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98388)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98388/logs/1140)|dotnet/runtime#79061|
|[98378](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98378)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98378/logs/927)|dotnet/runtime#79026|
|[98353](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98353)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98353/logs/680)|dotnet/runtime#78995|
|[98326](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98326)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98326/logs/618)|dotnet/runtime#79051|
|[98314](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98314)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98314/logs/954)|dotnet/runtime#79048|
|[98088](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98088)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98088/logs/1228)|dotnet/runtime#79024|
|[98229](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98229)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98229/logs/869)|dotnet/runtime#79046|
|[97948](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97948)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97948/logs/2232)|dotnet/runtime#79049|
|[98140](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98140)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98140/logs/715)||
|[98134](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98134)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98134/logs/707)||
|[98129](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98129)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98129/logs/843)||
|[97393](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97393)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97393/logs/2218)|dotnet/runtime#78087|
|[98103](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98103)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98103/logs/701)|dotnet/runtime#79007|
|[98080](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98080)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98080/logs/641)|dotnet/runtime#77130|
|[98056](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98056)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98056/logs/981)|dotnet/runtime#79057|
|[98062](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98062)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98062/logs/1082)||
|[98047](https://dev.azure.com/dnceng-public/public/_build/results?buildId=98047)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/98047/logs/486)|dotnet/runtime#74579|
|[96320](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96320)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96320/logs/2195)|dotnet/runtime#78852|
|[97985](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97985)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97985/logs/503)|dotnet/runtime#79019|
|[97981](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97981)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97981/logs/618)|dotnet/runtime#79051|
|[97922](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97922)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97922/logs/90)|dotnet/runtime#78829|
|[97905](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97905)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97905/logs/596)|dotnet/runtime#78915|
|[97870](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97870)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97870/logs/515)|dotnet/runtime#79044|
|[97862](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97862)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97862/logs/553)|dotnet/runtime#79044|
|[97841](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97841)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97841/logs/548)|dotnet/runtime#79042|
|[97253](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97253)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97253/logs/1280)|dotnet/runtime#79019|
|[97812](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97812)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97812/logs/67)|dotnet/runtime#79039|
|[97811](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97811)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97811/logs/612)|dotnet/runtime#79040|
|[97783](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97783)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97783/logs/70)|dotnet/runtime#79036|
|[97758](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97758)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97758/logs/944)|dotnet/runtime#79035|
|[97680](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97680)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97680/logs/836)|dotnet/runtime#78020|
|[97666](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97666)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97666/logs/966)|dotnet/runtime#78817|
|[97662](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97662)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97662/logs/545)|dotnet/runtime#78915|
|[97636](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97636)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97636/logs/1013)|dotnet/runtime#78850|
|[93797](https://dev.azure.com/dnceng-public/public/_build/results?buildId=93797)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/93797/logs/1264)|dotnet/runtime#78857|
|[96362](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96362)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96362/logs/1232)|dotnet/runtime#78646|
|[97564](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97564)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97564/logs/966)|dotnet/runtime#73215|
|[97520](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97520)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97520/logs/609)|dotnet/runtime#78915|
|[97491](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97491)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97491/logs/1016)|dotnet/runtime#79027|
|[97479](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97479)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97479/logs/409)||
|[97460](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97460)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97460/logs/1003)|dotnet/runtime#78974|
|[97453](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97453)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97453/logs/1008)||
|[97430](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97430)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97430/logs/504)|dotnet/runtime#78793|
|[97408](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97408)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97408/logs/1006)|dotnet/runtime#79026|
|[97404](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97404)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97404/logs/876)|dotnet/runtime#78861|
|[97383](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97383)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97383/logs/903)|dotnet/runtime#78863|
|[97377](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97377)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97377/logs/497)|dotnet/runtime#79025|
|[97373](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97373)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97373/logs/589)|dotnet/runtime#79024|
|[97359](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97359)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97359/logs/504)|dotnet/runtime#78793|
|[97337](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97337)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97337/logs/576)||
|[97325](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97325)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97325/logs/812)||
|[97308](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97308)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97308/logs/984)|dotnet/runtime#79021|
|[97206](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97206)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97206/logs/672)|dotnet/runtime#79013|
|[97194](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97194)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97194/logs/783)|dotnet/runtime#78860|
|[97200](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97200)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97200/logs/1229)|dotnet/runtime#78958|
|[97179](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97179)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97179/logs/538)|dotnet/runtime#78793|
|[97121](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97121)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97121/logs/1467)|dotnet/runtime#78996|
|[97089](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97089)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97089/logs/1049)|dotnet/runtime#79009|
|[97087](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97087)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97087/logs/117)|dotnet/runtime#79008|
|[97063](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97063)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97063/logs/118)|dotnet/runtime#78411|
|[97057](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97057)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97057/logs/1405)|dotnet/runtime#78943|
|[97043](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97043)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97043/logs/1033)|dotnet/runtime#79007|
|[97029](https://dev.azure.com/dnceng-public/public/_build/results?buildId=97029)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/97029/logs/959)|dotnet/runtime#78782|
|[96953](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96953)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96953/logs/1209)|dotnet/runtime#79002|
|[96960](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96960)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96960/logs/583)||
|[96946](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96946)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96946/logs/675)|dotnet/runtime#77130|
|[96951](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96951)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96951/logs/883)||
|[96884](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96884)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96884/logs/588)|dotnet/runtime#78946|
|[96848](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96848)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96848/logs/1039)|dotnet/runtime#78998|
|[96847](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96847)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96847/logs/79)|dotnet/runtime#78997|
|[96785](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96785)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96785/logs/360)||
|[96767](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96767)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96767/logs/84)|dotnet/runtime#78411|
|[96723](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96723)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96723/logs/845)|dotnet/runtime#78983|
|[96712](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96712)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96712/logs/1070)|dotnet/runtime#78943|
|[96704](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96704)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96704/logs/1303)||
|[96650](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96650)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96650/logs/914)|dotnet/runtime#78986|
|[96634](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96634)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96634/logs/1483)|dotnet/runtime#78984|
|[96573](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96573)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96573/logs/85)|dotnet/runtime#78982|
|[93237](https://dev.azure.com/dnceng-public/public/_build/results?buildId=93237)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/93237/logs/2258)|dotnet/runtime#78320|
|[96538](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96538)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96538/logs/947)|dotnet/runtime#78968|
|[96522](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96522)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96522/logs/1271)|dotnet/runtime#78974|
|[96495](https://dev.azure.com/dnceng-public/public/_build/results?buildId=96495)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/96495/logs/1026)|dotnet/runtime#78888|
Displaying 100 of 258 results
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|40|214|258|
<!--Known issue error report end -->

--- a/src/tasks/AppleAppBuilder/Xcode.cs
+++ b/src/tasks/AppleAppBuilder/Xcode.cs
@@ -528,8 +528,15 @@ internal sealed class Xcode
 
         Utils.RunProcess(Logger, "xcodebuild", args.ToString(), workingDir: Path.GetDirectoryName(xcodePrjPath));
 
-        string appPath = Path.Combine(Path.GetDirectoryName(xcodePrjPath)!, config + "-" + sdk,
-            Path.GetFileNameWithoutExtension(xcodePrjPath) + ".app");
+        string appDirectory = Path.Combine(Path.GetDirectoryName(xcodePrjPath)!, config + "-" + sdk);
+        if (!Directory.Exists(appDirectory))
+        {
+            // cmake 3.25.0 seems to have changed the output directory for MacCatalyst, move it back to the old format
+            string appDirectoryWithoutSdk = Path.Combine(Path.GetDirectoryName(xcodePrjPath)!, config);
+            Directory.Move(appDirectoryWithoutSdk, appDirectory);
+        }
+
+        string appPath = Path.Combine(appDirectory, Path.GetFileNameWithoutExtension(xcodePrjPath) + ".app");
 
         if (destination != null)
         {

